### PR TITLE
Fix linting warnings

### PR DESCRIPTION
**Pull request by luisas**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/create_tcoffee_template.nf [hunk 1]
```diff
@@ -17,11 +22,21 @@ process CREATE_TCOFFEETEMPLATE {
     """
     # Prep templates
     for structure in \$(ls *.pdb); do id=`echo \$structure| awk  {'gsub(".pdb", "", \$0); print'}`; echo -e ">"\$id "_P_" "\${id}" >>${prefix}_template.txt ; done
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        ubuntu: \$(cat /etc/lsb-release | grep "DISTRIB_DESCRIPTION" | awk '{gsub("DISTRIB_DESCRIPTION=", ""); print \$2}')
+    END_VERSIONS
```
Comment (JoseEspinosa) [inline modules/local/create_tcoffee_template.nf:29]: Please provide maybe bash version like here: https://github.com/nf-core/modules/blob/546e927b63fda8373eb6e23825599cbc29b0d94c/modules/nf-core/custom/tabulartogseagct/main.nf#L35-L38

File: modules/local/parse_sim.nf [hunk 2]
```diff
@@ -22,11 +27,21 @@ process PARSE_SIM {
 
     echo "id,perc_sim" > ${prefix}.sim_tot
     cat tmp | tr '\\n' ',' | awk 'gsub(/,\$/,x)' >>  ${prefix}.sim_tot
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        ubuntu: \$(cat /etc/lsb-release | grep "DISTRIB_DESCRIPTION" | awk '{gsub("DISTRIB_DESCRIPTION=", ""); print \$2}')
+    END_VERSIONS
```
Comment (JoseEspinosa) [inline modules/local/parse_sim.nf:34]: Please, provide sed version like [here](https://github.com/nf-core/modules/blob/546e927b63fda8373eb6e23825599cbc29b0d94c/modules/nf-core/igv/js/main.nf#L61-L64)

File: modules/local/prepare_shiny.nf [hunk 3]
```diff
@@ -22,11 +27,21 @@ process PREPARE_SHINY {
...
+    END_VERSIONS
     """
 
     stub:
     """
     touch shiny_data.csv
     touch shiny_app.R
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
```
Comment (JoseEspinosa) [inline modules/local/prepare_shiny.nf:43]: Bash version instead as above

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Nice, just ask for small stuff and add some text to readme, feel free to modify it as desired 😄

File: modules/local/create_tcoffee_template.nf (hunk 1 above)
Comment (luisas) [reply modules/local/create_tcoffee_template.nf]: Done!

File: modules/local/parse_sim.nf (hunk 2 above)
Comment (luisas) [reply modules/local/parse_sim.nf]: Done!

File: modules/local/prepare_shiny.nf (hunk 3 above)
Comment (luisas) [reply modules/local/prepare_shiny.nf:43]: Done!

File: README.md [hunk 4]
```diff
@@ -57,25 +57,45 @@ Available ALIGN methods:
...
+```
+
+`tree` is the tool used to build the tree.
+
+Arguments to the tree tool can be provided using `args_tree`.
+
+The `aligner` column contains the tool to run the alignment.
 
-<!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
+Finally, the arguments to the aligner tool can be set by using the `args_alginer` column.
```
Comment (JoseEspinosa) [inline README.md:96]:
```suggestion
Finally, the arguments to the aligner tool can be set by using the `args_aligner` column.
```

**Review by JoseEspinosa — APPROVED** [review]
Just a small typo from my previous review, otherwise awesome 🚀

**Pull request merged**